Install/install.sh

```diff
@@ -0,0 +1,192 @@
+  FIRST_TIME=1  # if this is a first time install
+
+  if [ "$SKIP_DEPS" = false ]; then
+    # install from apt
+    install_linux_packages
+
+    # configure the hardware
+    setup_hardware
+  else
+    echo -e "Skipping dependency installs, updating SlowMovie code only"
+  fi
+
+  if [ -d "${LOCAL_DIR}" ]; then
+    echo -e "Existing Install Found - Updating Repo"
+    cd $LOCAL_DIR
+    git fetch
+    git checkout $GIT_BRANCH
+    git pull
+
+    # go back to home directory
+    cd /home/pi/
+  else
+    echo -e "No Install Found - Cloning Repo"
+    git clone -b ${GIT_BRANCH} ${GIT_REPO} ${LOCAL_DIR}
+    FIRST_TIME=0
+  fi
+
+  if [ "$SKIP_DEPS" = false ]; then
+    # install any needed python packages
+    install_python_packages
+
+    # install any additional libraries we need to build manually
+    build_python_libraries
+  fi
+
+  cd $LOCAL_DIR
+  echo -e "SlowMovie install/update complete"
+
+  return $FIRST_TIME
+}
+
+function install_service(){
+  if [ -d "${LOCAL_DIR}" ]; then
+    cd $LOCAL_DIR
+    # install the service files and enable
+    sudo cp slowmovie.service /etc/systemd/system
+    sudo systemctl daemon-reload
+    sudo systemctl enable slowmovie
+
+    echo -e "SlowMovie service installed! Use ${YELLOW}sudo systemctl start slowmovie${RESET} to test"
+  else
+    echo -e "${RED}SlowMovie repo does not exist! Use option 1 - Install/Upgrade SlowMovie first${RESET}"
+  fi
+
+  # go back to home
+  cd /home/pi
+}
+
+function uninstall_service(){
+  if [ -f "/etc/systemd/system/slowmovie.service" ]; then
+    # stop if running and remove service files
+    sudo systemctl stop slowmovie
+    sudo systemctl disable slowmovie
+    sudo rm /etc/systemd/system/slowmovie.service
+    sudo systemctl daemon-reload
+
+    echo -e "SlowMovie service was successfully uninstalled"
+  else
+    echo -e "${RED}SlowMovie service is already uninstalled.${RESET}"
+  fi
+}
+
+# get any options
+while getopts ":r:b:si:h" arg; do
+    case "${arg}" in
+        r)
+          GIT_REPO=${OPTARG}
+        ;;
+        b)
+          GIT_BRANCH=${OPTARG}
+        ;;
+        s)
+          SKIP_DEPS=true
+        ;;
+        h)
+          echo "SlowMovie Install/Upgrade Script"
+          echo "Use this to install or upgrade your SlowMovie player"
+          echo "Advanced Usage:"
+          echo "[-r] specify a repo url"
+          echo "[-b] specify a repo branch"
+          echo "[-s] to skip dependency installs (update SlowMovie code only)"
+          exit 0
+        ;;
+    esac
+done
+
+# set the local directory
+LOCAL_DIR="/home/pi/$(basename $GIT_REPO)"
+
+echo -e "SlowMovie Repo set to ${YELLOW}${GIT_REPO}/${GIT_BRANCH}${RESET}"
+echo -e "Setting up in local directory ${YELLOW}${LOCAL_DIR}${RESET}"
+echo -e ""
+cd /home/pi/
+
+INSTALL_OPTION=$(whiptail --menu "Choose what you want to do." 0 0 0 1 "Install/Upgrade SlowMovie" 2 "Install SlowMovie Service" 3 "Uninstall SlowMovie Service" 4 "Exit" 3>&1 1>&2 2>&3)
+
+: ${INSTALL_OPTION:=4}
+
```

Comment:
```suggestion
```
Don't need this anymore.

Reply:
Going to keep it, if they hit __cancel__ the variable comes back blank.

Reply:
I forgot that -eq only works on numbers.